### PR TITLE
Update category

### PR DESCRIPTION
**Pull request by Capuche**
## Content
Update of how categories are handled. They are now stored in an Array as strings

### Minors changes
- Renamed challenge in the settings (there was the ancient name 'nice-form')
- Added a small seed
- Added content to README
- Removed custom fonts

### CODE REVIEW

File: app/assets/stylesheets/components/_card.scss [hunk 1]
```diff
@@ -22,15 +22,15 @@
...
     background: image-url('ruby.svg'), white;
   }
   
-  &.javascript{
+  &.JavaScript{
     background: image-url('javascript.svg'), white;
   }
   
-  &.css{
+  &.CSS{
```
Comment (Eschults) [inline app/assets/stylesheets/components/_card.scss]: Let's downcase the class names 👌

File: app/models/snippet.rb [hunk 2]
```diff
@@ -1,6 +1,8 @@
 class Snippet < ApplicationRecord
-  belongs_to :category
+  CATEGORIES = ["Ruby", "JavaScript", "CSS"]
+
   has_many :snippet_tags
   has_many :tags, through: :snippet_tags
   validates :title, presence: true
+  validates_inclusion_of :category, in: CATEGORIES
```
Comment (Eschults) [inline app/models/snippet.rb]: [`validates_inclusion_of`](https://apidock.com/rails/ActiveModel/Validations/ClassMethods/validates_inclusion_of) is deprecated, let's use the [`inclusion`](https://guides.rubyonrails.org/active_record_validations.html#inclusion) option 👇 

```ruby
validates :category, inclusion: { in: CATEGORIES }
```

File: app/views/snippets/_snippet.html.erb [hunk 3]
```diff
@@ -1,4 +1,4 @@
-<div class="col-sm-12 col-md-6 offset-md-3 card preview <%= snippet.category.style %> <%= closed ||= '' %>">
+<div class="col-sm-12 col-md-6 offset-md-3 card preview <%= snippet.category %> <%= closed ||= '' %>">
```
Comment (Eschults) [inline app/views/snippets/_snippet.html.erb]:
```suggestion
<div class="col-sm-12 col-md-6 offset-md-3 card preview <%= snippet.category.downcase %> <%= closed ||= '' %>">
```

**Comment by Capuche** [comment]
Hi @Eschults :wave:

I just fixed the delete button (missing `dependant :destroy`).
All the requested updated are commited

**Comment by Eschults** [comment]
👍

**Pull request merged**